src/main/java/org/threeten/extra/chrono/InternationalFixedEra.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Given we're completely aligned with `CE`, you might just want to use the standard ISO era (from `LocalDate`), then just restrict input as appropriate.